.github/workflows/update-website.yml

```diff
@@ -0,0 +1,25 @@
+name: Update website
+
+on:
+  push:
+    branches:
+      - master
+    paths:
+      - "integrations/integrations.json"
+
+jobs:
+  trigger:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Trigger netdata/website update-integrations workflow
+        env:
+          TARGET_REPO: "netdata/website"
+          WORKFLOW_FILE: "update-integrations.yml"
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
Because this is a cross-repo dispatch, this token will not work for this invocation. You’ll need the `NETDATABOT_GITHUB_TOKEN` secret instead (that _should_ work, provided the netdatabot user is listed for appropriate access to the target repo).

Reply:
Changed that